examples/ao_host.js

```diff
@@ -67,7 +67,7 @@ host.on('error', (err) => {
   debug('error: %s', err)
 })
 
-host.once('ws2:auth:success', async () => {
+host.once('ready', async () => {
```

Comment:
there is one event `auth:success` will have to check if there is `ws2:auth:success`

Reply:
but yeah .. we aren't starting starting AO until we get `ready`